frontend/src/views/pages/inventory/InventoryAdd.vue

```diff
@@ -0,0 +1,226 @@
+        <h6 class="modal-title">Vehicle added successfully!</h6>
+        <CButtonClose @click="showingModal = false" />
+      </template>
+      <template #footer>
+        <CButton @click="showingModal = false" color="success">Yes</CButton>
+        <router-link to="/inventory"><CButton color="danger">No</CButton></router-link>
+      </template>
+    </CModal>
+  </div>
+</template>
+
+<script>
+const axios = require('axios');
+const { containsPermissions } = require("../../../utils/index");
+import MaskedInput from 'vue-text-mask'
+
+export default {
+  name: 'InventoryAdd',
+  data() {
+    return {
+      dealership: null,
+      vehicleProperties: null,
+      errorMessage: null,
+      showingModal: false
+    }
+  },
+  methods: {
+    submitForm(form) {
+      let properties = {};
+      this.vehicleProperties.forEach(property => {
+        if (!form.target.elements[property.key]) {
+          properties[property.key] = null;
+        } else {
+          if (property.inputType == "List") {
+            const itemString = form.target.elements[property.key].value;
+            const items = itemString.split(";");
+            properties[property.key] = items;
+          } else if (property.inputType == "Date") {
+            properties[property.key] = form.target.elements[property.key].value
+          } else if (property.inputType == "Number") {
+            properties[property.key] = form.target.elements[property.key].value
+          } else if (property.inputType == "Currency") {
+            properties[property.key] = form.target.elements[property.key].value
+          } else {
+            properties[property.key] = form.target.elements[property.key].value
+          }
+        }
+      })
+      
+      // submit the vehicle
+      this.submitVehicle(properties);
+    },
+    submitVehicle(properties) {
+      const body = {
+        dealership: this.$route.params.dealershipId,
+        properties
+      };
+      console.log(body);
+      axios({
+        method: "POST",
+        url: `${this.$store.state.api}/inventory`,
+        headers: {
+          Authorization: `Bearer ${this.$store.state.auth.token}`,
+        },
+        data: body
+      })
+      .then((response) => {
+        if (response.data.success) {
+          //TODO: show modal asking to enter a new vehicle or go back to the inventory
+          // for now, just refresh the page
+          this.$refs.vehicleForm.reset();
+          this.showingModal = true;
+        }
+      })
+      .catch((error) => {
+        console.log(error);
+        this.showErrorMessage('Unable to create the vehicle. Make sure the information was entered correctly.');
+      });
+    },
+    showErrorMessage(message) {
+      this.errorMessage = message;
+      setTimeout(() => {
+        this.errorMessage = null;
+      }, 5000)
+    },
+    userHasPermissions(...permissions) {
+      // console.log(permissions);
+      // console.log(containsPermissions(permissions));
```

Comment:
Remove commented code

Reply:
Will create a pull request next sprint that will clean up all the console.logs and error cases for API requests